must/must.go

```diff
@@ -0,0 +1,41 @@
+package must
+
+import "strings"
+
+func Must[T any](obj T, err error) T {
+	if err != nil {
+		panic(err)
+	}
+
+	return obj
+}
+
+func MustBeOk[T any](value T, ok bool, args ...error) T {
+	if !ok {
+		panic(BuildErrorMessage("must be ok", args))
+	}
+	return value
+}
+
+func MustNotBeOk[T any](value T, ok bool, args ...error) T {
+	if ok {
+		panic(BuildErrorMessage("must not be ok", args))
+	}
+	return value
+}
+
+func BuildErrorMessage(defaultErrorMessage string, args []error) string {
```

Comment:
1. Instead of taking `[]error` as an argument, u can make it a variable to make function calls easier (`args ...error`)
2. No needs to be **public**